resources/cheatsheets/index.html

```diff
@@ -23,25 +23,27 @@ <h1 class="display-alpha m-y-0">Git Cheat Sheets</h1>
-            <li><a href="{{ site.baseurl }}/downloads/zh_CN/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> Chinese</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/fr/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> French</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/fr/github-git-cheat-sheet.pdf"><span class="octicon octicon-cloud-download"></span> French (PDF)</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> English</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/github-git-cheat-sheet.pdf"><span class="octicon octicon-cloud-download"></span> English (PDF)</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/de/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> German</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/it/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> Italian</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/ja/github-git-cheat-sheet"><span class="octicon octicon-cloud-download"></span> Japanese</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/ja/github-git-cheat-sheet.pdf"><span class="octicon octicon-cloud-download"></span> Japanese (PDF)</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/kr/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> Korean</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/pt_BR/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> Portuguese (Brazil)</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/pt_BR/github-git-cheat-sheet.pdf"><span class="octicon octicon-cloud-download"></span> Portuguese (Brazil) (PDF)</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/pt_PT/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> Portuguese (Portugal)</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/pt_PT/github-git-cheat-sheet.pdf"><span class="octicon octicon-cloud-download"></span> Portuguese (Portugal) (PDF)</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/sk/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> Slovak</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/es_ES/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download"></span> Spanish</a></li>
-            <li><a href="{{ site.baseurl }}/downloads/es_ES/github-git-cheat-sheet.pdf"><span class="octicon octicon-cloud-download"></span> Spanish (PDF)</a></li>
+            <li><a href="{{ site.baseurl }}/downloads/de/github-git-cheat-sheet/"><span class="octicon octicon-cloud-download German"></span> Deutsche</a></li>
```

Comment:
Unfortunately we have a spelling error here :abcd:

It's **`Deutsch`** not `Deutsche`.